edvart/report.py

```diff
@@ -35,6 +37,14 @@ class EmptyReportWarning(UserWarning):
     """Warning raised when a report contains no sections."""
 
 
+class ExportDataMode(str, Enum):
```

Comment:
Yes, `StrEnum` would be better, but it is only available with Python 3.11+

Reply:
I modified the example accidentally. Reverted to the original example.
I will look a bit more into the enum. It would be nice to be able to specify it with just a string (e.g. `"embed"`) rather than having to import the enum.

Reply:
I found a package `strenum` which adds this functionality. Using the package for python 3.10 and older; using the std. lib. version in Python 3.11+.